src/main/java/com/kodilla/ecommercee/services/UserService.java

```diff
@@ -24,8 +26,8 @@ public User blockUserById(final Long userId) throws UserNotFoundException {
     }
 
     public User generateKey(final String username) throws UserNotFoundException {
-        Random rand = new Random();
-        long randomUserKey = rand.nextInt(90000) + 10000;
+        SecureRandom rand = new SecureRandom();
+        long randomUserKey = (long)rand.nextInt(90000) + (long)10000;
```

Comment:
nie lepiej zastosować zamiast rzutowań nextLong?

Reply:
wyskakuje wtedy problem z rangem i ilością liczb (trzeba by stworzyc wtedy osobna klase i metode, co wydluzyloby niepotrzebnie kod), jak po mergu wyskoczy code smell, pomysle jeszcze nad tym, na razie to rozwiazanie eliminuje problemy ;)